helix-view/src/theme.rs

```diff
@@ -77,6 +93,14 @@ impl Loader {
         names
     }
 
+    pub fn default_theme(&self, true_color: bool) -> Theme {
+        if true_color {
+            self.default()
+        } else {
+            self.base16_default()
+        }
+    }
```

Comment:
I wish we could use base16 since that is the terminal color so the color of helix uses the same colorscheme the terminal uses. But that's something that @archseer have to decide.